name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/DkbPDFExtractor.java

```diff
@@ -35,10 +35,10 @@ public DkbPDFExtractor(Client client)
         addInvestmentEarningTransaction();
         addRemoveTransaction();
         addTransferOutTransaction();
+        addEarning27KstgTransaction();
         addParticipationcertificateEarningTransaction();
         addParticipationcertificateRefundTransaction();
         addInvestmentPayoutTransaction();
-        addBuyTransactionFundsSavingsPlan();
```

Comment:
Hast Du diese Implementierung mit Absicht gelöscht? Wenn ja, warum? Wenn ich das richtig sehe, hat @ZfT2 die vor 6 Wochen hinzugefügt.

Das ist auch der Grund dafür warum der Build fehlschlägt. Der Test ist nämlich nicht gelöscht und schlägt jetzt fehl. Du kannst die Tests übrigens auch direkt in der Eclipse IDE ausführen, wenn Du ~/portfolio-app/eclipse öffnest und dann per Rechtsklick "Run As..." auswählst.